crates/valence_inventory/src/lib.rs

```diff
@@ -405,6 +409,17 @@ impl HeldItem {
     pub fn slot(&self) -> u16 {
         self.held_item_slot
     }
+
+    pub fn set_slot(&mut self, slot: u16) {
+        // temp
+        assert!(
+            (36..=44).contains(&slot),
```

Comment:
The packet seems to be using 0..8 https://wiki.vg/Protocol#Set_Held_Item

Reply:
I am aware that the packet uses 0..8, the reason why i'm not using it here is to keep it consistent with the slot() return value, which is 36..44, and changing that is out of scope of this PR imo and backwards-compat breaking. (see line 684 where it gets converted)

Reply:
yea right, weird, but not the scope of the pr